php/Terminus/Commands/CliCommand.php

```diff
@@ -90,9 +90,10 @@ public function completions($args, $assoc_args) {
   public function console($args, $assoc_args) {
     $user = Session::getUser();
     if (isset($assoc_args['site'])) {
-      $sitename = $assoc_args['site'];
-      $site_id  = $this->sitesCache->findId($sitename);
-      $site     = new Site($site_id);
+      $sitename  = $assoc_args['site'];
```

Comment:
Since this `--site=<site>` arg is optional, we should use the Input helper here or try/catch.